tests/fixtures.py

```diff
@@ -8,18 +11,59 @@
 @fixture(name="dvc_repo_session", scope="session")
 def fix_dvc_repo_session() -> DvcRepo:
     with tempfile.TemporaryDirectory() as dir:
-        subprocess.check_call(["git", "init"], cwd=dir)
-        print(dir)
-        dvc = DvcRepo.init(dir, subdir=True)
+        this_dir = dir+"/fix_dvc_repo_session"
+        subprocess.check_call(["git", "init"], cwd=this_dir)
+        dvc = DvcRepo.init(this_dir, subdir=True)
+        prev_dir = os.getcwd()
+        os.chdir(this_dir)
+
         yield dvc
-        dvc.close()
 
+        dvc.close()
+        os.chdir(prev_dir)
 
 @fixture(name="dvc_repo")
 def fix_dvc_repo(dvc_repo_session) -> DvcRepo:
-    subprocess.check_call(
-        f"cp -r {dvc_repo_session.root_dir} {dvc_repo_session.root_dir}_1",
-        shell=True,
-    )
-    dvc_repo_session.root_dir += "_1"
-    return dvc_repo_session
+    dvc_repo = copy.deepcopy(dvc_repo_session)
+    dvc_repo.root_dir += "_dvc_repo"
+    shutil.copytree(dvc_repo_session.root_dir, dvc_repo.root_dir)
+    prev_dir = os.getcwd()
+    os.chdir(dvc_repo.root_dir)
+
+    yield dvc_repo
+    
+    os.chdir(prev_dir)
+
+@fixture(name="empty_repo_session", scope="session")
```

Comment:
oops -- our call starts ... better would have been to `stash` and make the fix to the previous pr, then pop stash and merge. But easy to do....